.github/workflows/_push_image.yaml

```diff
@@ -0,0 +1,78 @@
+name: Push images
+
+# This is a reusable workflow that is called from any other workflow that needs 
+# to build and push the configurator image to DockerHub
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Version as is going to use for the binary'
+        type: string
+        required: true
+      image-tag:
+        description: 'Tag as is going to use for the image'
+        type: string
+        required: true
+      image-overwrite-latest:
+        description: 'If enabled, an extra tag `latest` will be pushed apart from the defined tag'
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  push-image:
+    name: Build and Push images to Docker Hub
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: newrelic/newrelic-prometheus-configurator
+      DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with make build-multiarch
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Build integration
+        env:
+          TAG: ${{ inputs.version }}
+        run: make build-multiarch
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
```

Comment:
One day we should rename these secrets 🤣

Reply:
you never known , perhaps is anticipating the future?